tgui/packages/tgui/interfaces/PlaneMasterDebug.tsx

```diff
@@ -382,10 +383,7 @@ export class PlaneMasterDebug extends Component {
     const { plane_info, mob_name } = data;
     const [showAdd, setShowAdd] = useLocalState('showAdd', false);
 
-    const [connectSources, setConnectSouces] = useLocalState<AssocConnected>(
-      'connectionSources',
-      {},
-    );
+    const [connectSources, setConnectSouces] = useState<AssocConnected>({});
```

Comment:
this and 932 are strange. does this throw a TS error?

Reply:
It does.

![JyOcX2WXmC](https://github.com/tgstation/tgstation/assets/5195984/36c7b76b-f1f6-431b-9e52-7c91d69b70d3)

Reply:
wait 😲 this is using useLocalState within a class component!
this should be left alone. why wouldn't they use this.state here??

Reply:
Reverted with https://github.com/tgstation/tgstation/pull/80459/commits/796956a956bda82faa16413b9cd16361d43008a9